### PR TITLE
W-11758424 add helm steps to license key

### PR DESCRIPTION
**Pull request by IsaacEldridge**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/_partials/include-install-tasks.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
 // tag::licenseKey[]
 == Insert the Mule License Key
 
-If you didn’t add the Mule license key during install, you can add it using the `rtfctl` command line utility.
+If you didn’t add the Mule license key during install, you can add it using the `rtfctl` command line utility or Helm.
 
-After the installation has completed succesfully, insert the Mule license key.
+Before you install the license key, Base64 encode it.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-install-tasks.adoc]: <Channeling Kim> you should probably avoid verbing nouns...

Is there a way to say this differently? "Encode to Base64 format"?

Comment (hannanelson) [reply modules/ROOT/pages/_partials/include-install-tasks.adoc]: Or hyphenate? "Base64-encode"

File: modules/ROOT/pages/_partials/include-install-tasks.adoc [hunk 2]
```diff
@@ -32,7 +32,9 @@ BASE64_ENCODED_LICENSE=$(base64 -w0 license.lic)
 $BASE64_ENCODED_LICENSE=[convert]::ToBase64String((Get-Content -path "license.lic" -Encoding byte))
 ----
 
-. On the controller node acting as the leader during installation (the installer node), use the `rtfctl` utility with the Base64 value of your license key: 
+=== Apply the License Key Using `rtfctl`
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-install-tasks.adoc]: remove formatting from heading

File: modules/ROOT/pages/_partials/include-install-tasks.adoc [hunk 3]
```diff
@@ -45,6 +47,14 @@ rtfctl apply mule-license $BASE64_ENCODED_LICENSE
 ----
 rtfctl get mule-license
 ----
+
+=== Apply the License Key Using Helm
+To apply the license using Helm, run the following command:
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-install-tasks.adoc:53]: Do we need a blank line between heading and text?

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/_partials/include-install-tasks.adoc (hunk 1 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/_partials/include-install-tasks.adoc]: oh good point

**Review by hannanelson — APPROVED** [review]

**Pull request merged**